components/atoms/Accordion/Accordion.js

```diff
@@ -0,0 +1,53 @@
+import propTypes from "prop-types";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronRight,
+  faChevronDown,
+} from "@fortawesome/free-solid-svg-icons";
+
+export function Accordion({ title, summary, description }) {
+  const [openFlag, setOpenFlag] = useState(false);
+  function openSummary() {
+    setOpenFlag(!openFlag);
+  }
+
+  return (
+    <div className="border-gray-300 flex border border-b-2 border-r-2 mb-2 p-8 shadow-lg">
+      <div className="w-8  min-w-8 relative top-5">
+        {openFlag ? (
+          <FontAwesomeIcon icon={faChevronRight} size="2x" color="red" />
+        ) : (
+          <FontAwesomeIcon icon={faChevronDown} size="2x" color="red" />
```

Comment:
I found when alternating between these 2 icons when clicking made the icon snap slightly to the left when showing the right facing arrow. If you replace this with

```JavaScript
<FontAwesomeIcon icon={faChevronRight}  rotation={90} size="2x" color="red" />
```
the icon's position is more static.